docs/modules/releasenotes/pages/changelog.adoc

```diff
@@ -4,31 +4,31 @@
 
 [[releasenotes-changelog-29.0.4]]
 
-==== Release 29.0.4
+== Release 29.0.4
```

Comment:
@RangerRick I think this is something we probably need to fix in our JIRA changelog template, right?